### DISCUSSION TITLE
Implement soft delete record ?

### DISCUSSION THREAD

**Discussion by Tilican** [pocketbase/pocketbase#4502] [Q&A]
Hey,

Thanks again for this exceptional project!
We all know the case where the customer says, no it's not me it's deleted all by itself!
So it's all in the title :)

Thanks

**Comment by dschissler** [comment]
I think that you'd have to implement a custom API point with PB as a framework using Javascript or Golang.  You would need to check a status field for not deleted in all API filters and queries.  In the custom API call you would simply save the status field to deleted.
